spec/support/capybara.rb

```diff
@@ -1,21 +1,21 @@
-require 'capybara/rspec'
-require 'capybara-screenshot/rspec'
-require 'chromedriver-helper'
```

Comment:
will requiring `webdrivers` include these capybara helpers, or are they being added somewhere else??

Reply:
Ah, I copied the file from `engine_storefront` and missed that change. Fixed.